src/main/java/uk/gov/hmcts/cmc/claimstore/utils/CaseDetailsConverter.java

```diff
@@ -15,14 +19,35 @@
 
     private final CaseMapper caseMapper;
     private final JsonMapper jsonMapper;
+    private final int intentionToProceedDeadline;
```

Comment:
This definitely is not the right place for this field. I would not expect any change in this utility class for your story.

Reply:
@daveoh We should not have any business logic as part of our CCD mappers, so there is no point of injecting WorkingDayIndicator in ccd-adapter. The mappers should be simply mapping to/from. 

I would expect this business logic to be part of the action/trigger to calculate this deadline set this field in `Claim` then mapper will have simple job to map this and save in CCD. When we will read this claim back from CCD, we will simply map it back from CCD to our `Claim` object.

Reply:
@sabahirfan This has been discussed, but requires a definition release but we have a time constraint to get this released today. Are you happy to waive this for the short term?

Reply:
@daveoh lets discuss that offline